### DISCUSSION TITLE
Code button only inserts 1 ` mark

### DISCUSSION THREAD

**Discussion by hucker75** [community/community#62279] [Discussions]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The <> button above this text I'm typing inserts a single ` either side of the cursor, yet the instructions state there should be three, on a seperate line.

I get 'pasted code' instead of:

[three ticks]
pasted code
[three ticks]

This makes linefeeds break in the pasted code.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @hucker75! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by hucker75** [comment]
Thanks, I couldn't work out where to put it!

**Comment by dakshithadissanayaka** [comment]
Hi there,

Thank you for reporting this issue. I can see that the code block formatting is not working as expected. The button should insert three backticks (```) on a separate line, but it is only inserting one backtick (`). This is causing the code block to be formatted incorrectly, and linefeeds are breaking in the pasted code.

I have created a bug report for this issue and will be working on a fix. In the meantime, you can use the following workaround to format your code blocks correctly:

1. Manually type three backticks (```) on a separate line before and after your code block.
2. Remove the "pasted code" label from the code block.

I apologize for any inconvenience this may cause.

Here is an example of how to use the workaround:

```
```
The <> button above this text I'm typing inserts a single ` either side of the cursor, yet the instructions state there should be three, on a seperate line.
I get 'pasted code' instead of:
```
```
This makes linefeeds break in the pasted code.
```
```

Thank you for your understanding.
